packages/components/src/internal/components/search/SampleFinderSection.tsx

```diff
@@ -0,0 +1,183 @@
+        setShowFilterModal(true);
+    }, [filterCards]);
+
+    const onFilterDelete = useCallback((index: number) => {
+        const newFilterCards = [...filterCards];
+        newFilterCards.splice(index, 1);
+        setFilterCards(newFilterCards);
+    }, [filterCards]);
+
+    const onFilterClose = () => {
+        setShowFilterModal(false);
+    };
+
+    const onFind = useCallback((schemaQuery: SchemaQuery, filterArray: Filter.IFilter[])  => {
+        onFilterClose();
+        let newFilterCards = [...filterCards];
+        newFilterCards.push({
+            schemaQuery,
+            filterArray,
+            entityDataType: chosenEntityType,
+            index: filterCards.length,
+            onAdd: onAddEntity,
+            onDelete: onFilterDelete,
+        });
+        setFilterCards(newFilterCards);
+    }, [setFilterCards, filterCards, onFilterEdit, onFilterDelete, chosenEntityType]);
+
+    return (
+        <Section title={SAMPLE_FINDER_TITLE} caption={SAMPLE_FINDER_CAPTION} context={(
+            <SampleFinderHeaderButtons parentEntityDataTypes={parentEntityDataTypes} onAddEntity={onAddEntity}/>
+        )}
+        >
+            {filterCards.length == 0 ?
+                <>
+                    <FilterCards className="empty" cards={parentEntityDataTypes.map((entityDataType) => ({
+                        entityDataType,
+                        onAdd: onAddEntity
+                    }))} />
+                    <div className="filter-hint">{getFinderStartText(parentEntityDataTypes)}</div>
+                </>
+                :
+                <>
+                    <FilterCards cards={filterCards} />
+                    <SampleFinderSamples
+                        {...gridProps}
+                        cards={filterCards}
+                    />
+                </>
+            }
+            {showFilterModal && (
+                <EntityFieldFilterModal
+                    onCancel={onFilterClose}
+                    entityDataType={chosenEntityType}
+                    onFind={onFind}
+                />
+            )}
+        </Section>
+    )
+});
+
+interface SampleFinderSamplesProps extends SampleFinderSamplesGridProps {
+    cards: FilterCardProps[];
+}
+
+export const SampleFinderSamplesImpl: FC<SampleFinderSamplesProps & InjectedQueryModels> = memo(props => {
+    const { actions, queryModels } = props;
+
+    return(
+        <>
+            <h4>Filtering and sample-type tabs coming soon...</h4>
+            <SamplesTabbedGridPanel
+                {...props}
+                withTitle={false}
+                asPanel={false}
+                actions={actions}
+                queryModels={queryModels}
+                tabbedGridPanelProps={{
+                    alwaysShowTabs: true,
+                    advancedExportOptions: SAMPLE_DATA_EXPORT_CONFIG,
+                }}
+            />
+        </>
+    )
+});
+
+// exported for jest testing
```

Comment:
I don't see this actually getting used in the jest testing. I assume it doesn't need to be exported.

Reply:
Removed for now.  Might get tested separately later.